<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI_Dialog_Welcome</class>
 <widget class="QDialog" name="UI_Dialog_Welcome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>597</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Willkommen zu learn Java</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="l_text">
     <property name="styleSheet">
      <string notr="true">font-size:16pt; color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>Herzlich Willkommen</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="l_image">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>144</width>
       <height>85</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1435</width>
       <height>850</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Screenshot_0.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pB_next_page">
       <property name="locale">
        <locale language="German" country="Germany"/>
       </property>
       <property name="text">
        <string>NÃ¤chste Seite</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
